### PR TITLE
ASB-274/275 implement search box

### PR DESCRIPTION
**Pull request by pk2453**

### CODE REVIEW

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/controllers/projects/documents.js [hunk 1]
```diff
@@ -46,6 +47,8 @@ function renderData(req, res, params, response) {
 }
 
 exports.getAboutTheApplication = async (req, res) => {
+  const dt = moment('2019-04-22', 'YYYY-MM-DD').format('YYYY-MM-DD');
```
Comment (cpcundill) [inline packages/forms-web-app/src/controllers/projects/documents.js:50]: Assuming this and the line below need to be removed and was just used for some debugging?

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/views/layouts/project-3-columns.njk [hunk 2]
```diff
@@ -6,11 +6,18 @@
...
+      <div class="govuk-grid-column-three-quarters">
+        <div class="govuk-grid-row govuk-!-margin-left-3 govuk-!-margin-right-3">
+          {% block resultSec %}{% endblock %}
+        </div>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            {% block contentHeader %}{% endblock %}
+          </div>
+          <div class="govuk-grid-column-one-third">
+            {% block filterSection %}{% endblock %}
```
Comment (cpcundill) [inline packages/forms-web-app/src/views/layouts/project-3-columns.njk:18]: This should still be `columnThree`

File: packages/forms-web-app/src/views/layouts/project-3-columns.njk [hunk 3]
```diff
@@ -6,11 +6,18 @@
...
-      </div>
-      <div class="govuk-grid-column-one-quarter">
-        {% block columnThree %}{% endblock %}
+      <div class="govuk-grid-column-three-quarters">
+        <div class="govuk-grid-row govuk-!-margin-left-3 govuk-!-margin-right-3">
+          {% block resultSec %}{% endblock %}
+        </div>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            {% block contentHeader %}{% endblock %}
```
Comment (cpcundill) [inline packages/forms-web-app/src/views/layouts/project-3-columns.njk:15]: This should be `columnTwo`

File: packages/forms-web-app/src/views/layouts/project-3-columns.njk [hunk 4]
```diff
@@ -6,11 +6,18 @@
...
         {% include "layouts/vertical-tabs.njk" %}
       </div>
-      <div class="govuk-grid-column-one-half">
-        {% block columnTwo %}{% endblock %}
-      </div>
-      <div class="govuk-grid-column-one-quarter">
-        {% block columnThree %}{% endblock %}
+      <div class="govuk-grid-column-three-quarters">
+        <div class="govuk-grid-row govuk-!-margin-left-3 govuk-!-margin-right-3">
+          {% block resultSec %}{% endblock %}
```
[the remark this answers is not in the document]
Comment (cpcundill) [reply packages/forms-web-app/src/views/layouts/project-3-columns.njk:11]: This should become `contentHeader`

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/views/layouts/project-3-columns.njk (hunk 3 above)
Comment (pk2453) [reply packages/forms-web-app/src/views/layouts/project-3-columns.njk]: All are changed

**Review by cpcundill — APPROVED** [review]

**Pull request merged**